src/views/NoticeOfWithdrawal.vue

```diff
@@ -432,7 +431,7 @@ export default class NoticeOfWithdrawal extends Mixins(CommonMixin, DateMixin, F
       // this is the id of filing being withdrawn, and of THIS filing
       // if filingID is 0, this is a new filing
       // otherwise it's a draft filing
-      this.filingToBeWithdrawn = Number(this.$route.query.filingToBeWithdrawn) || null
+      this.filingToBeWithdrawn = +this.$route.query.filingToBeWithdrawn
```

Comment:
Should you add a check that this is provided (same as filingId)? See lines 437-440.